### ISSUE TITLE
UD Validator issue: deprel is advmod but upos is not ADV

### ISSUE THREAD

**Issue by NathanD38** [IAHLT/UD_Hebrew#5]
@amir-zeldes

It seems that the validator does not accept an `advmod` deprel to a token with a upos other than `ADV`.
But I thought that fixed MWE are the exception, where it is possible to find an `ADP` as the first token, 
e.g.,  כמו כן has an `ADP` as its first token,

**Comment by amir-zeldes** [comment]
Yes, that's exactly right, the validator will allow `advmod` on the first token of a subgraph that has all other tokens attached as `fixed` to that first token, regardless of POS. Can you post the sentence where you're seeing a validation error?

**Comment by NathanD38** [comment]
Sorry. I am not sure this was an error from the validator, or from an automatic message created by Yael's validator.
I'm waiting to hear from her or from Noam regarding this.

This is one of many sentences, in which this error is present in the QA as reason for review.

`sent = עם זאת, בואזיז לא מראה סימני דאגה: “החזרה למשרדים לא משדרת לנו שהתופעה של עבודה מרחוק נעלמת, להיפך”, הוא אומר.
`
The error is: `deprel is advmod but upos is not ADV in line 1`

> 
**1	עם	עם	ADP	ADP	_	6	advmod	_	_
2	זאת	זה	PRON	PRON	Gender=Fem|Number=Sing|Person=3|PronType=Dem	1	fixed	_	_**
3	,	,	PUNCT	PUNCT	_	2	punct	_	_
4	בואזיז	בואזיז	PROPN	PROPN	_	6	nsubj	_	_
5	לא	לא	ADV	ADV	Polarity=Neg	6	advmod	_	_
6	מראה	הראה	VERB	VERB	Gender=Fem|HebBinyan=HIFIL|Number=Sing|Person=3|Tense=Pres|VerbForm=Part|Voice=Act	0	root	_	_
7	סימני	סימן	NOUN	NOUN	Definite=Cons|Gender=Masc|Number=Plur	6	obj	_	_
8	דאגה	דאגה	NOUN	NOUN	Gender=Fem|Number=Sing	7	compound	_	_
9	:	:	PUNCT	PUNCT	_	16	punct	_	_
10	“	“	PUNCT	PUNCT	_	16	punct	_	_
11	ה	ה	DET	DET	Definite=Def|PronType=Art	12	det	_	_
12	חזרה	חזרה	NOUN	NOUN	Gender=Fem|Number=Sing	16	nsubj	_	_
13	ל	ל	ADP	ADP	Definite=Def	14	case	_	_
14	משרדים	משרד	NOUN	NOUN	Gender=Masc|Number=Plur	12	nmod	_	_
15	לא	לא	ADV	ADV	Polarity=Neg	16	advmod	_	_
16	משדרת	שידר	VERB	VERB	Gender=Fem|HebBinyan=PIEL|Number=Sing|Person=3|Tense=Pres|VerbForm=Part|Voice=Act	6	parataxis	_	_
17	ל	ל	ADP	ADP	Definite=Def	18	case	_	_
18	נו	הוא	PRON	PRON	Gender=UnSpec|Number=Plur|Person=1|PronType=Prs	16	obl	_	_
19	ש	ש	SCONJ	SCONJ	_	26	mark	_	_
20	ה	ה	DET	DET	Definite=Def|PronType=Art	21	det	_	_
21	תופעה	תופעה	NOUN	NOUN	Gender=Fem|Number=Sing	26	nsubj	_	_
22	של	של	ADP	ADP	Case=Gen	23	case	_	_
23	עבודה	עבודה	NOUN	NOUN	Gender=Fem|Number=Sing	21	nmod:poss	_	_
24	מ	מ	ADP	ADP	_	25	case	_	_
25	רחוק	רחוק	ADJ	ADJ	Gender=Masc|Number=Sing	23	nmod	_	_
26	נעלמת	נעלם	VERB	VERB	Gender=Fem|HebBinyan=NIFAL|Number=Sing|Person=3|Tense=Pres|VerbForm=Part|Voice=Mid	16	ccomp	_	_
27	,	,	PUNCT	PUNCT	_	28	punct	_	_
28	להיפך	להיפך	ADV	ADV	Xtra=Junk	26	advmod	_	_
29	”	”	PUNCT	PUNCT	_	16	punct	_	_
30	,	,	PUNCT	PUNCT	_	6	punct	_	_
31	הוא	הוא	PRON	PRON	Gender=Masc|Number=Sing|Person=3|PronType=Prs	32	nsubj	_	_
32	אומר	אמר	VERB	VERB	Gender=Masc|HebBinyan=PAAL|Number=Sing|Person=3|Tense=Pres|VerbForm=Part|Voice=Act	6	parataxis	_	_
33	.	.	PUNCT	PUNCT	_	6	punct	_	_

**Comment by NathanD38** [comment]
Yael recommended to ignore this message for the moment, 
until her code can differentiate between fixed MWE and non-fixed expressions with mismatched deprel and upos.

**Comment by amir-zeldes** [comment]
OK, yeah, that's not from the official UD validator, which I recommend using in addition to other checks:

https://github.com/universalDependencies/tools

> python validate.py --lang he somefile.conllu
